packages/main-ui/src/pages/components/footer.rs

```diff
@@ -89,3 +91,69 @@ translate! {
         en: "• Sitemap",
     },
 }
+
+#[component]
+pub fn Footer(lang: Language) -> Element {
+    let tr: FooterTranslate = translate(&lang);
+    let responsive_service: ResponsiveService = use_context();
+
+    rsx! {
+        Responsive {
+            if responsive_service.width() > 1200.0 {
```

Comment:
don't use conditional rendering it will distrub SEO.
you should use visibility and adjust class.